src/components/Link.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { routeTo } from 'redux-router-kit';
+
+const LinkView = ({ targetTo, text, path }) =>
```

Comment:
In the other components I used the naming scheme `SomethingComponent`. I like `SomethingView` better! :D Let's move to this naming scheme. (As a separate PR)